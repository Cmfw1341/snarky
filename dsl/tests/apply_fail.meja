let d : _ = fun (f : 'b -> 'b, x : unit) => {
  f 15;
  f x;
};
