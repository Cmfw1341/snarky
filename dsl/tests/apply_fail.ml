let (d : (unit -> unit) -> unit -> '_a17) =
 fun (f : unit -> unit) (x : unit) -> f 15 ; f x
