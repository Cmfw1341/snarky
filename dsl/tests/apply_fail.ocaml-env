type char
type float
type int
type string
type unit = unit
