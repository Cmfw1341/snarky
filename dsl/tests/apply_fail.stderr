Type error: Cannot unify 'int'1 and 'unit'2.
File "tests/apply_fail.meja", line 3, characters 2-5:
Error: Type error: Cannot unify 'unit -> unit'12 and 'unit -> '_a19'20, types 
'int'1 and 'unit'13 are incompatable.

